galley/pkg/meshconfig/cache.go

```diff
@@ -45,6 +45,13 @@ type FsCache struct {
 var _ Cache = &FsCache{}
 var _ io.Closer = &FsCache{}
 
+var DefaultMeshConfig = func() v1alpha1.MeshConfig {
```

Comment:
duplicate with that in **`galley/pkg/config/meshcfg/defaults.go`**

Reply:
@ozevren made that function not set these ingress settings. Not sure of the context, I think he would be able to answer intended behavior best? ref https://github.com/istio/istio/pull/17142

Or we could just change the default for everything not special case

Reply:
pkg/config/... is the new pipeline that I am going to be introducing enabling soon. This change needs to be reflected there as well.